recipes/xbbg/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xbbg" %}
+{% set version = "0.7.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/alpha-xone/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Why change to GitHub from PyPI?

Reply:
`requirements.txt` is missing in PyPI source and the package doesn't build, so I switch to GitHub